README.md

```diff
@@ -188,16 +188,23 @@ Here is a snippet of a `Dockerfile` to add in tini (be sure to use it in `CMD` o
 
 ```Dockerfile
 # grab tini for signal processing and zombie killing
-ENV TINI_VERSION v0.9.0
-RUN set -x \
-	&& curl -fSL "https://github.com/krallin/tini/releases/download/$TINI_VERSION/tini" -o /usr/local/bin/tini \
-	&& curl -fSL "https://github.com/krallin/tini/releases/download/$TINI_VERSION/tini.asc" -o /usr/local/bin/tini.asc \
+ENV TINI_VERSION v0.18.0
+RUN set -ex \
```

Comment:
We should probably go all the way if we're updating this and do `set -eux;` and use semicolons instead since their `set -e` behavior is much more reliable (since `&&` creates a "complex expression" and the `set -e` rules become less predictable, especially with things like `for` loops embedded).